frontend/src/metabase/ui/components/inputs/Checkbox/Checkbox.styled.tsx

```diff
@@ -35,6 +35,7 @@ export const getCheckboxOverrides = (): MantineThemeOverride["components"] => ({
           },
         },
       },
+      body: { alignItems: "center" },
```

Comment:
this is a global change, but it seemed like an oversight: checkboxes should be center-aligned with their text by default, right @kdoh ?

Reply:
don't forget that if some checkboxes have flex-direction=row, I believe this will have an unexpected effect